osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -154,6 +155,7 @@ private void clearComments()
         {
             currentPage = 1;
             deletedChildrenPlaceholder.DeletedCount.Value = 0;
+            moreButton.Show();
```

Comment:
why was this added in this PR? it doesn't look like a `Hide` call was added anywhere.

Reply:
It was missing because moreButton can be hidden at line 187 (`moreButton.FadeTo(response.HasMore ? 1 : 0);`)

Reply:
Anyway I changed it a little to make more sense